src/Symfony/Component/Notifier/Bridge/FreeMobile/README.md

```diff
@@ -1,16 +1,16 @@
 Free Mobile Notifier
 ====================
 
-Provides Free Mobile integration for Symfony Notifier.
+Provides [Free Mobile](https://free-mobile.fr) integration for Symfony Notifier.
+
 This provider allows you to receive an SMS notification
 on your personal mobile number.
 
 DSN example
 -----------
 
-```
-// .env file
-FREE_MOBILE_DSN=freemobile://LOGIN:PASSWORD@default?phone=PHONE
+```ini
+freemobile://LOGIN:PASSWORD@default?phone=PHONE
```

Comment:
i do not understand this change? this was taken from the recipes so that the readme is selfexplained

Reply:
It's an example of the DSN, it does not need to be assigned to an environment variable and the bridge itself does not "know" anything about a "recipe", so I think the environment variable is not important in this context